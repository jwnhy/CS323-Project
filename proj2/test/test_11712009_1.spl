int main(int a, int b) {
    int f = a[0];
}