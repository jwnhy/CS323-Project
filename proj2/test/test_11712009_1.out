Error type 10 at Line 2: Not Array.
Error type 5 at Line 2: Two sides of assignment have inconsistent type.
